src/SamplesApp/UITests.Shared/Windows_UI_StartScreen/JumpListTestsViewModel.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text;
+using System.Windows.Input;
+using Windows.UI.Core;
+using Windows.UI.StartScreen;
+using UITests.Shared.Helpers;
+using Uno.UI.Common;
+using Uno.UI.Samples.UITests.Helpers;
+
+namespace UITests.Shared.Windows_UI_StartScreen
+{
+	public class JumpListTestsViewModel : ViewModelBase
+	{
+		private ICommand _loadCurrentCommand;
+		private JumpList _jumpList;
+		private JumpListItem _selectedItem;
+		private ObservableCollection<JumpListItem> _items = new ObservableCollection<JumpListItem>();
+		private ICommand _removeItemCommand;
+		private ICommand _addItemCommand;
+		private NewJumpListItem _newItem = new NewJumpListItem();
+
+		public JumpListTestsViewModel(CoreDispatcher dispatcher) : base(dispatcher)
+		{
+			IsSupported = JumpList.IsSupported();
+		}
+
+		public bool IsSupported { get; }
+
+		public bool IsLoaded => _jumpList != null;
+
+		public bool ItemSelected => SelectedItem != null;
+
+		public ObservableCollection<JumpListItem> Items
+		{
+			get => _items;
+			private set
+			{
+				_items = value;
+                RaisePropertyChanged();
+			}
+		}
+
+		public JumpListItem SelectedItem
+		{
+			get => _selectedItem;
+			set
+			{
+				_selectedItem = value;
+				RaisePropertyChanged(nameof(ItemSelected));
+			}
+		}
+
+		public NewJumpListItem NewItem
+		{
+			get => _newItem;
+			set
+			{
+				_newItem = value;
+				RaisePropertyChanged(nameof(NewItem));
+			}
+		}
+
+		public ICommand AddItemCommand =>
+			_addItemCommand ?? (_addItemCommand = new DelegateCommand(AddItemAsync));
```

Comment:
There's a `base.GetOrCreateCommand()` you can use instead of DelegateCommand...
<https://github.com/unoplatform/uno/blob/master/src/SamplesApp/UITests.Shared/Helpers/ViewModelBase.cs#L55>

I think we should have an overload to support async methods.

Reply:
Cool! will do

Reply:
Done 😊